news/_posts/translation/2021-01-08-do-you-speak-prestashop-december-2020.md

```diff
@@ -0,0 +1,121 @@
+
+### :two_hearts: Special thanks to newcomers 
+ 
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straight away. For that reason, we would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated [Jiří VALTR](https://crowdin.com/profile/gvg), [Store](https://crowdin.com/profile/helabasa), [Kiana Vafaei](https://crowdin.com/profile/k1994v), [Paolo Battistella](https://crowdin.com/profile/paolo.battistella), [Alex Darcy](https://crowdin.com/profile/nachtzehrer), and [Emin çaki](https://crowdin.com/profile/emincaki)!
+
+## A few stats
+ 
+* **29 members** were active on the project this month.
+* A total of **43,689 words have been translated** and **38,579 validated**.
+* All this in **24 different languages**.
+ 
+Thank you for your involvement!
+
+## Top contributors
+ 
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and PrestaShop can't thank you enough for your dedication! Here are the 10 most active translators and proofreaders for December 2020.
+ 
+#### Top 10 translators in December:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+| 1. | Rijad Osmanovic (rijado) | Bosnian | 21,090
+| 2. | Jiří VALTR (GVG) | Czech | 9,832
+| 3. | Tatu Wikman (tswfi) | Finnish | 2,085
+| 4. | Benjamin Gantikow (bbbenjie) | German | 2,000
+| 5. | ‫דניאל שפירא‬‎ (danielshapiro9) | Hebrew | 1,473
+| 6. | Stamatis (breezer) | Greek | 1,254
+| 7. | Odelia Yechiel (OdeChan) | Hebrew | 1,050
+| 8. | Federico Ferreri (fferreri) | Spanish, Argentina | 926
+| 9. | Zoran Tejic (zotamal) | Serbian (Latin) | 778
+| 10. | Sretko Devič (Chico) | Slovenian | 571
+ 
+
+#### Top 10 proofreaders in December:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | Rijad Osmanovic (rijado) | Bosnian | 21,273
+| 2. | Jiří VALTR (GVG) | Czech | 8,759
+| 3. | Benjamin Gantikow (bbbenjie) | German | 2,527
+| 4. | ‫דניאל שפירא‬‎ (danielshapiro9) | Hebrew | 2,155
+| 5. | Stamatis (breezer) | Greek | 1,332
+| 6. | Gerardas (gerardas) | Lithuanian | 786
+| 7. | Zoran Tejic (zotamal) | Serbian (Latin) | 785
+| 8. | Sretko Devič (Chico) | Slovenian | 571
+| 9. | Rauno Riikman (weaver) | Finnish | 103
+| 10. | webdvl (megashopba) | Slovak | 82
+ 
+Congrats, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who's been contributing to our translation project thanks to the [Translators page](http://translators.prestashop.com/).
+
+ 
+## Complete translations
+ 
+#### Fully translated languages
+ 
+At the end of December, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **12 languages**:
+ 
+| **Bosnian** | Chinese traditional | Danish | English | **French** | **Greek** | **Italian**| Portuguese, Brazil | **Serbian** | **Slovak** | **Slovene** | Swedish |
```

Comment:
```suggestion
| **Bosnian** | Chinese traditional | Danish | English | **French** | **Greek** | **Italian** | Portuguese, Brazil | **Serbian** | **Slovak** | **Slovene** | Swedish |
```

Reply:
We used to create tables grouping 4 by 4 languages to have a more legible overview, just like for the top 10 contributors. How those 12 languages will appear on Build, aligned on a single row?

Reply:
Yes, I was confused because last time I was told that they should be aligned on a single row to form a table. 😅 And I don't know why the table does not appear in the preview like in the final publication... :( I'm going to do a screenshot for next time. 🤣

Reply:
Ah? Why should they be aligned on a single row?

Reply:
I'm not sure 🤔 I can't find the message but it was from @matks

Reply:
But maybe it was because I had only 5 languages at the time...

Reply:
When we click on "View" on this file we have a preview by GitHub of what the page will look like. Here the preview is https://github.com/PrestaShop/prestashop.github.io/blob/f7db4eb81221aa9a1a35ffe0f55dcdbb615db759/news/_posts/translation/2021-01-08-do-you-speak-prestashop-december-2020.md

On the preview the languages appear on a single row **but that might not be the final result** because GitHub _attempts_ to render it but the rendering is not 100% done by GitHub. So final rendering and preview rendering can differ.

If you are sure it will be rendered correctly, go ahead 😉